src/lesson_21/Main.java

```diff
@@ -0,0 +1,40 @@
+package lesson_21;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+/*
+Создать программу для чтения номеров контрактов из файла.
+Неповторяющиеся номера контрактов следует сохранить в структуру данных ключ-значение,
+где ключ - это номер документа, а значение - это объекта класса Document со всей контекстной
+информацией (номер документа, дата создания и другая информация).
+Структуру данных ключ-значение вывести на экран.
+ */
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+        FileReader file = new FileReader("C:\\Users\\ADMIN\\IdeaProjects\\Vasilisa\\src\\NumberDoc");
+        Scanner scan = new Scanner(file);
+        HashSet<String> numberDocument = new HashSet<>();
```

Comment:
It's better to use interfaces to define variables.
Eg. `Set<String> numberDocument = new HashSet<>();`